pkg/apis/core/v1alpha1/zz_generated.kubebuilder.go

```diff
@@ -281,9 +281,11 @@ var (
 				Plural: "federatedclusters",
 			},
 			Scope: "Cluster",
+			Subresources: &v1beta1.CustomResourceSubresources{
```

Comment:
If you use kubebuilder v0.1.12, you will have the Subersources https://github.com/kubernetes-sigs/kubebuilder/commit/bc21162c8199b45d2d191f9a0e78c171925dc786

Which version of kubebuilder are you using? I have upgraded cluster-registry to use kubebuilder at https://github.com/kubernetes/cluster-registry/pull/244